pkgs/games/osu-lazer/bin.nix

```
@@ -1,31 +1,34 @@
-{ lib
-, stdenv
-, fetchurl
-, fetchzip
-, appimageTools
-, makeWrapper
+{
+  lib,
+  stdenv,
+  fetchurl,
+  fetchzip,
+  appimageTools,
+  makeWrapper,
 }:
 
 let
   pname = "osu-lazer-bin";
   version = "2024.906.2";
 
-  src = {
-    aarch64-darwin = fetchzip {
-      url = "https://github.com/ppy/osu/releases/download/${version}/osu.app.Apple.Silicon.zip";
-      hash = "sha256-KyvC8gEqZvXMATxS2513X0WdlR7nF8tHS4R/TPFrHao=";
-      stripRoot = false;
-    };
-    x86_64-darwin = fetchzip {
-      url = "https://github.com/ppy/osu/releases/download/${version}/osu.app.Intel.zip";
-      hash = "sha256-ToxDZHL59YPmybvB9tsiOnFEd+FJJE4mNMfaK6btYKo=";
-      stripRoot = false;
-    };
-    x86_64-linux = fetchurl {
-      url = "https://github.com/ppy/osu/releases/download/${version}/osu.AppImage";
-      hash = "sha256-zQnR3KwlE1gTWH8f+GDRBsc7Whfn9XpT1D/NLg5TtrU=";
-    };
-  }.${stdenv.system} or (throw "${pname}-${version}: ${stdenv.system} is unsupported.");
+  src =
+    {
+      aarch64-darwin = fetchzip {
+        url = "https://github.com/ppy/osu/releases/download/${version}/osu.app.Apple.Silicon.zip";
+        hash = "sha256-KyvC8gEqZvXMATxS2513X0WdlR7nF8tHS4R/TPFrHao=";
+        stripRoot = false;
+      };
+      x86_64-darwin = fetchzip {
+        url = "https://github.com/ppy/osu/releases/download/${version}/osu.app.Intel.zip";
+        hash = "sha256-ToxDZHL59YPmybvB9tsiOnFEd+FJJE4mNMfaK6btYKo=";
+        stripRoot = false;
+      };
+      x86_64-linux = fetchurl {
+        url = "https://github.com/ppy/osu/releases/download/${version}/osu.AppImage";
+        hash = "sha256-zQnR3KwlE1gTWH8f+GDRBsc7Whfn9XpT1D/NLg5TtrU=";
+      };
+    }
+    .${stdenv.system} or (throw "${pname}-${version}: ${stdenv.system} is unsupported.");
```

```suggestion
    .${stdenv.system} or (throw "osu-lazer-bin: ${stdenv.system} is unsupported.");
```
pname is not necessarily the attr which is important here.